src/common/Database/DatabaseEnv.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is part of the WarheadCore Project. See AUTHORS file for Copyright information
```

Comment:
please update this. It's ok to mention Warhead but make this standard as the other files